README.md

```diff
@@ -745,9 +752,16 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
   Useful for code relying on `Function.prototype.name`. See also: the `keep_fnames`
   [compress option](#compress-options).
 
-- `eval` (default `false`). Pass `true` to mangle names visible in scopes
-  where `eval` or `with` are used.
+- `cache` (default `false`). Boolean for cache determination.
```

Comment:
@kzc please provide the copy you would like to see and I will update. Again it was like a cat and mouse game chasing docs.